libs/server/src/lib/utils/utils.ts

```diff
@@ -290,3 +290,33 @@ export function getPrimitiveValue(value: any): string | undefined {
     return undefined;
   }
 }
+
+function renameProperty(obj: any, from: string, to: string) {
+  Object.keys(obj).forEach(k => {
```

Comment:
you don't need to loop, right? you aren't using `k` for anything.

Reply:
gah, sorry yeah. will fix.